scripts/update/Updater.php

```diff
@@ -715,5 +715,35 @@ public function update($initialVersion) {
 
             $this->setVersion('5.31.0');
         }
+        
+        if ($this->isVersion('5.31.0')) {
+
+            $extension = \common_ext_ExtensionsManager::singleton()->getExtensionById('taoQtiTest');
+
+            $config = $extension->getConfig('testRunner');
+            $config['shortcuts'] = [
+                'calculator' => [
+                    'toggle' => 'C',
+                ],
+                'zoom' => [
+                    'in' => 'I',
+                    'out' => 'O'
+                ],
+                'comment' => [
+                    'toggle' => 'A',
+                ],
+                'itemThemeSwitcher' => [
+                    'toggle' => 'T',
+                ],
+                'review' => [
+                    'toggle' => 'R',
+                    'flag' => 'M'
+                ]
+            ];
+
```

Comment:
I'm wondering about conflicts with a client extension if the client update happens before, won't we lose its customisations ?